### DISCUSSION TITLE
how to hide cards in tweets and timeline ??

### DISCUSSION THREAD

**Discussion by deepghuge36** [andrewsuzuki/react-twitter-widgets#55] [Q&A] [answered]
` <Tweet
                            tweetId={elementDetails.tweetId}
                            options={{
                                theme: elementDetails.isDark ? 'dark' : 'light',
                                width: elementDetails.styling.width - 20,
                                height: elementDetails.styling.height - 30,
                            }}
                            renderError={() => (
                                <p>
                                    Could not load Tweet Please Check the Tweet
                                    URL or Tweet ID
                                </p>
                            )}
                        />`

**Comment by andrewsuzuki** [comment, marked as answer]
This library is just a light wrapper around the twitter embed library, which injects an iframe. So you can't really modify the contents of that iframe. Perhaps contact the twitter developers if you want the option added.
